src/main/java/mate/academy/springboot/swagger/util/SortUtil.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public class SortUtil {
+    private static final int DIRECTION_INDEX = 1;
+    private static final int PARAM_INDEX = 0;
+    private static final String SEPARATOR_ATTRIBUTES = ":";
+    private static final String SEPARATOR_PARAMS = ";";
+    private static List<Sort.Order> orders;
+
+    public static List<Sort.Order> sort(String sortBy) {
```

Comment:
```suggestion
    public List<Sort.Order> sort(String sortBy) {
```
don't use static if it's possible, it's a bad practice